Orange/evaluation/testing.py

```diff
@@ -276,7 +276,7 @@ def get_augmented_data(self, model_names, include_attrs=True, include_prediction
 
         # add fold info
         if self.folds is not None:
-            new_meta_attr.append(DiscreteVariable(name="Fold", values=[i+1 for i, s in enumerate(self.folds)]))
+            new_meta_attr.append(DiscreteVariable(name="Fold", values=[str(i+1) for i, s in enumerate(self.folds)]))
```

Comment:
@kernc 
`# the second alternative below is the fastest:
[i+1 for i, s in enumerate(self.folds)]  # 0.957
[i+1 for i, _ in enumerate(self.folds)]  # 0.916
[str(i) for i, _ in enumerate(self.folds, 1)]  # 2.527
[str(i) for i in range(1, len(self.folds) + 1)]  # 2.417
np.arange(1, len(self.folds) + 1, dtype=str).tolist()  # does not work`